app/lib/services/voip/mockCall.ts

```
@@ -1,8 +1,26 @@
 import type { CallState, IClientMediaCall } from '@rocket.chat/media-signaling';
 
-import Navigation from '../../navigation/appNavigation';
 import { useCallStore } from './useCallStore';
 
+jest.mock('../../navigation/appNavigation', () => ({
```

ESLint

[error] 'jest' is not defined.

(no-undef)